src/drivers/distance_sensor/sf1xx/sf1xx.cpp

```diff
@@ -857,13 +858,32 @@ info()
 static void
 sf1xx_usage()
 {
-	PX4_INFO("usage: sf1xx command [options]");
-	PX4_INFO("options:");
-	PX4_INFO("\t-b --bus i2cbus (%d)", SF1XX_BUS_DEFAULT);
-	PX4_INFO("\t-a --all");
-	PX4_INFO("\t-R --rotation (%d)", distance_sensor_s::ROTATION_DOWNWARD_FACING);
-	PX4_INFO("command:");
-	PX4_INFO("\tstart|stop|test|reset|info");
+	PRINT_MODULE_DESCRIPTION(
+		R"DESCR_STR(
+### Description
+
+I2C bus driver for Lightware SFxx series LIDAR rangefinders: SF10/a, SF10/b, SF10/c, SF11/c, SF/LW20.
+
+Setup/usage information: https://docs.px4.io/en/sensor/sfxx_lidar.html
+
+### Examples
+
+Attempt to start driver on any bus (start on bus where first sensor found).
+$ sf1xx start -a
+Stop driver
+$ sf1xx stop
+)DESCR_STR");
+
+	PRINT_MODULE_USAGE_NAME("sf1xx", "driver");
```

Comment:
@bkueng This is correct and builds, but I was wondering if it makes sense for us to have subcategories of drivers - ie "distance_sensor" in line with the folder structure and display them as a subcategory of driver in the sidebar? 
The reason is that there are lots of subcategories and driver page would be hard to navigate with all of them. 
I tried this and it failed with missing category, so before pursuing further I wanted to know whether you see benefit in the division.

It might also be worth building the categorisation into the API - ie adding an option subcategory. I did try this using current API and new option, but was told the category does not exist.
```
PRINT_MODULE_USAGE_NAME("sf1xx", "driver", "optional_sub_category");
```

Reply:
We can add a subcategory, but since it's a C API we cannot have default parameters. So it should be one of the following:
- add a `PRINT_MODULE_USAGE_DRIVER(name, subcategory)`
- add a `PRINT_MODULE_USAGE_SUBCATEGORY(subcategory)`